internal/component/database_observability/mysql/collector/schema_table.go

```diff
@@ -195,19 +196,25 @@ func (c *SchemaTable) extractSchema(ctx context.Context) error {
 				break
 			}
 
-			var table string
+			var tableName, tableType string
 			var createTime, updateTime time.Time
-			if err := rs.Scan(&table, &createTime, &updateTime); err != nil {
+			if err := rs.Scan(&tableName, &tableType, &createTime, &updateTime); err != nil {
```

Comment:
I'd expect the `selectTableName` query to be updated to fetch table type?

Reply:
Seems like a victim of resolving conflicts. Let me look thanks